src/components/Visualization/index.js

```diff
@@ -216,13 +216,13 @@ class VisualizationView extends React.Component {
                             let graphQueryParams = {};
                             let resetFilters = false;
 
-                            if(configuration.key) {
+                            if(configuration.graph === 'HeatmapGraph') {
```

Comment:
This should be handled in a configuration for the HeatmapGraph! Similarly to how we have done for table